packages/js/src/dashboard/components/table-widget.js

```diff
@@ -62,7 +62,7 @@ export const TableWidget = ( { title, children } ) => {
 			<Title as="h3" size="2" className="yst-text-slate-900 yst-font-medium">
 				{ title }
 			</Title>
-			<div className="yst-overflow-auto">
+			<div className="yst-overflow-auto yst-pe-1">
```

Comment:
Is this to compensate for the lack of padding in the UI library table now? 🤯

Reply:
moved it to the button, this is due to the focus ring id cutoff by the overflow hidden.